src/pages/overview/pdf-services-api/howtos/linearize-pdf.md

```diff
@@ -32,27 +32,33 @@ Please refer the [API usage guide](../api-usage.md) to understand how to use our
        public static void main(String[] args) {
     
            try {
-               // Initial setup, create credentials instance.
-               Credentials credentials = Credentials.servicePrincipalCredentialsBuilder()
-                    .withClientId("PDF_SERVICES_CLIENT_ID")
-                    .withClientSecret("PDF_SERVICES_CLIENT_SECRET")
-                    .build();
+                // Initial setup, create credentials instance
+                Credentials credentials = new ServicePrincipalCredentials(
+                        System.getenv("PDF_SERVICES_CLIENT_ID"),
+                        System.getenv("PDF_SERVICES_CLIENT_SECRET"));
     
-               // Create an ExecutionContext using credentials and create a new operation instance.
-               ExecutionContext executionContext = ExecutionContext.create(credentials);
-               LinearizePDFOperation linearizePDFOperation = LinearizePDFOperation.createNew();
+                // Creates a PDF Services instance
+                PDFServices pdfServices = new PDFServices(credentials);
     
-               // Set operation input from a source file.
-               FileRef source = FileRef.createFromLocalFile("src/main/resources/linearizePDFInput.pdf");
-               linearizePDFOperation.setInput(source);
+                // Creates an asset from source file and upload
+                InputStream inputStream = Files.newInputStream(new File("src/main/resources/linearizePDFInput.pdf").toPath());
+                Asset asset = pdfServices.upload(inputStream, PDFServicesMediaType.PDF.getMediaType());
+    
+                // Creates a new job instance
+                LinearizePDFJob linearizePDFJob = new LinearizePDFJob(asset);
     
-               // Execute the operation
-               FileRef result = linearizePDFOperation.execute(executionContext);
+                // Submits the job and gets the job result
+                String location = pdfServices.submit(linearizePDFJob);
+                PDFServicesResponse<LinearizePDFResult> pdfServicesResponse = pdfServices.getJobResult(location, LinearizePDFResult.class);
     
-               // Save the result at the specified location
-               result.saveAs("output/linearizePDFOutput.pdf");
+                // Get content from the resulting asset(s)
+                Asset resultAsset = pdfServicesResponse.getResult().getAsset();
+                StreamAsset streamAsset = pdfServices.getContent(resultAsset);
     
-           } catch (ServiceApiException | IOException | SdkException | ServiceUsageException ex) {
+                // Creates an output stream and copy stream asset's content to it
```

Comment:
*copies. if creates is being used ( same valid for all the places )

Reply:
As discussed, IG this is from already reviewed samples.